draft-ietf-mops-streaming-opcons.md

```diff
@@ -187,6 +193,47 @@ under peak load?
 
 Provisioning issues for caching systems?
 
+## Personalization and Advertizing
+
+Some of this fits better under adaptive bitrate, but most of it seems
+to tie to caching and bandwidth.  Maybe refactor or move as the other
+sections are developed.
+
+A simple model of video playback can be described as a video stream
+consumer, a buffer, and a transport mechanism that fills the buffer.
+The consumption rate is fairly static and is represented by the
+content bitrate.  The size of the buffer is also commonly a fixed
+size.  The fill process needs to be at least fast enough to ensure
+that the buffer is never empty, however it also can have significant
+complexity when things like personalization or ad workflows are
+introduced.
+
+The challenges in filling the buffer in a timely way fall into two
+broad categories: 1. content selection and 2. content variation.
+Content selection comprises all of the steps needed to determine which
+content variation to offer the client.  Content variation is the
+number of content options that exist at any given selection point.  A
+common example easily visualized is Adaptive Bitrate, described in
+more detail below.  The mechanism used to select the bitrate is part
+of the content selection, and the content variation are all of the
+different bitrate renditions.
+
+A similar but more complex case is the use of an ad selection service
```

Comment:
I think this paragraph is a good start, but we probably need more explanation about "targeting" in this context, and it might be beneficial to list sections with the different kinds of ads, with their pros and cons (server-side vs client-side ad insertion, and maybe a pointer to the iab.net specs for VPAID or something as an example of the kind of automated system for personalized ad insertion that is often used, maybe a mention of the variety of providers using realtime bidding to determine the selected content, and the drivers of the timing parameters that can have an impact on user experience?